contrib/mac/framework/julia-config.jl

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env julia
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+const options =
+[
+    "--cflags",
+    "--ldflags",
+    "--ldlibs"
+];
```

Comment:
that sounds preferable. why do we need the flexibility to name libjulia something different here (aside from the -debug difference)? external embedding consumers who want to do something like that are going to be fairly rare, I would think?